modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CsharpNetcoreFunctionsServerCodegen.java

```diff
@@ -403,6 +403,15 @@ public OperationsMap postProcessOperationsWithModels(OperationsMap objs, List<Mo
             if (operations != null) {
                 List<CodegenOperation> ops = operations.getOperation();
                 for (CodegenOperation operation : ops) {
+
+
+                    for (CodegenParameter param : operation.allParams) {
+                        if (param.isPathParam)
+                        {
+                            param.paramName = unCamelize(param.paramName);
```

Comment:
@justinjxzhang instead of "unCamlize" the parameter name, what about using `baseName` of the parameter mustache tag in the template instead?